lib/sasl/vsn.mk

```diff
@@ -1 +1 @@
-SASL_VSN = 4.2
+SASL_VSN = 4.2.1
```

Comment:
Versions are automatically updated based on ticket states, so you should not update the version manually.

Reply:
Right thanks, done.